cluster-autoscaler/simulator/cluster.go

```diff
@@ -29,6 +30,14 @@ import (
 	"github.com/golang/glog"
 )
 
+var (
```

Comment:
How about having those flags in the main file? I really don't like having flags spread across multiple files.